GoogleForCreators/web-stories-wp
# Ensure `amp-video` v0.1 is being used

## Context



See #8364 for details.

We were accidentally adding v1.0 of `amp-video` instead of v0.1.

## Summary



Updates PHP AMP sanitization to use `amp-video` v0.1 whenever missing.

## Relevant Technical Choices



Ports over change and tests from https://github.com/ampproject/amp-wp/pull/6303

Also updates `@link` references to point to v2.1.3 of the AMP plugin to make cross-referencing easier.

## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Insert Tenor GIF
2. Preview story
3. Verify GIF (video) is visible and playing

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8364
